rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -1295,6 +1295,16 @@ stream {
             {{ range $errCode := $location.CustomHTTPErrors }}
             error_page {{ $errCode }} = @custom_{{ $location.DefaultBackendUpstreamName }}_{{ $errCode }};{{ end }}
 
+            {{ if (eq $location.BackendProtocol "FCGI") }}
+            include /etc/nginx/fastcgi_params;
```

Comment:
This file doesn't exist (we do not copy unnecessary files from the default directory.) 
We need to copy /usr/local/openresty/nginx/conf/fastcgi_params to /etc/nginx